src/main/java/io/github/hypixel_api_wrapper/wrapper/games/bedwars/HypixelBedWarsCoreModes.java

```diff
@@ -0,0 +1,16 @@
+package io.github.hypixel_api_wrapper.wrapper.games.bedwars;
+
+import org.json.JSONObject;
+
+public class HypixelBedWarsCoreModes extends AbstractHypixelBedWars {
+
+    public HypixelBedWarsCoreModes(JSONObject stats) {
+        super(stats);
+    }
+
+    @Override
+    String prefix() {
+        return null;
```

Comment:
Instead of leaving a null here, wouldn't an empty string make it less error-prone?

Reply:
Fixed in #33, which is an extension of this PR